x-pack/plugins/observability_ai_assistant/server/functions/esql/system_message.txt

```diff
@@ -198,6 +206,8 @@ ROW a = "2023-01-23T12:15:00.000Z - some text - 127.0.0.1"
 ```esql
 FROM employees
 | WHERE first_name LIKE "?b*"
+| EVAL doc_count = COUNT(*)
```

Comment:
this is an agg no?

Reply:
yes! thank you for the catch, i corrected it